src/execution_plan/ops/op.h

```diff
@@ -103,6 +106,22 @@ void OpBase_AddVolatileRecord(OpBase *op, const Record r);
  * may be released. */
 void OpBase_RemoveVolatileRecords(OpBase *op);
 
+/* Mark alias as being modified by operation.
+ * Returns the ID associated with alias. */
+int OpBase_Modifies(OpBase *op, const char *alias);
+
+/* Mark alias as being projected by operation.
+ * Returns the ID associated with alias. */
+int OpBase_Projects(OpBase *op, const char *alias);
```

Comment:
Can you please elaborate why do we need this function?

Reply:
Oh, good catch - this function was deleted, I forgot to update the header file.

Reply:
👍